templates/layout/base.html

```diff
@@ -24,14 +24,13 @@
             window.lazySizesConfig = window.lazySizesConfig || {};
             window.lazySizesConfig.loadMode = 1;
         </script>
-        <script async src="{{cdn 'assets/dist/theme-bundle.head_async.js'}}"></script>
+        <script async src="{{cdn 'assets/dist/theme-bundle.head_async.js' resourceHint='preload' as='script'}}"></script>
         
-        {{getFontsCollection font-display='block'}}
+        {{getFontsCollection font-display='block' resourceHint='preload'}}
```

Comment:
this update is not needed. this helper and the `stylesheet` one will, by default, produce preload hints.